ports/unix/unix_mphal.c

```diff
@@ -141,6 +141,18 @@ static int call_dupterm_read(size_t idx) {
 }
 #endif
 
+static bool inputAvailable()
```

Comment:
Maybe this should be `inputAvailable(void)` to be consistent with other functions?

Reply:
Yes, and also `input_available(void)`, please check the code style from the surrounding functions. Also for the rest: opening brace on same line.